src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -129,6 +131,9 @@ protected override void ConfigureOptions()
             this.Add("shadowcopy", "Shadow copy test files",
                 v => ShadowCopyFiles = v != null);
 
+            this.Add("loaduserprofile", "Load user profile on test runner processes",
```

Comment:
Ah, yes. English is not my first language so you are probably right. I will update it